engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCache.scala

```diff
@@ -86,7 +86,7 @@ class CallCache(database: CallCachingSqlDatabase) {
   private def cachedResultOption(callCachingJoinOption: Option[CallCachingJoin]): Option[CachedResult] = {
     callCachingJoinOption map { callCachingJoin =>
       CachedResult(
-        callCachingJoin.callCachingEntry.returnCode,
+        callCachingJoin.callCachingEntry,
```

Comment:
ToL: At this point, the classes `engine.….CachedResult` and `database.….CallCachingJoin` are _effectively_ the same. One part of me likes this separation, the other feels like the database class could just be passed to the actor as is, removing the need for this mapping function.